warehouse/models/intermediate/payments/int_payments__refunds_deduped.sql

```diff
@@ -0,0 +1,150 @@
+{{ config(materialized = "table") }}
+
+WITH
+
+micropayments_table_refunds AS (
+    SELECT
+
+        aggregation_id,
+        micropayment_id,
+        participant_id,
+        customer_id,
+        ABS(charge_amount) AS refund_amount,
+        EXTRACT(DATE FROM transaction_time AT TIME ZONE "America/Los_Angeles") AS transaction_date,
+
+        -- add columns that we want to preserve from refunds table after union as null strings
+        SAFE_CAST(NULL AS STRING) AS refund_id,
+        SAFE_CAST(NULL AS STRING) AS settlement_id,
+        SAFE_CAST(NULL AS STRING) AS retrieval_reference_number,
+        SAFE_CAST(NULL AS STRING) AS coalesced_id,
+        SAFE_CAST(NULL AS NUMERIC) AS transaction_amount,
+        SAFE_CAST(NULL AS NUMERIC) AS proposed_amount,
+        SAFE_CAST(NULL AS STRING) AS status,
+        SAFE_CAST(NULL AS STRING) AS initiator,
+        SAFE_CAST(NULL AS STRING) AS reason,
+        SAFE_CAST(NULL AS STRING) AS approval_status,
+        SAFE_CAST(NULL AS STRING) AS issuer,
+        SAFE_CAST(NULL AS STRING) AS issuer_comment,
+        SAFE_CAST(NULL AS TIMESTAMP) AS created_time,
+        SAFE_CAST(NULL AS TIMESTAMP) AS approved_time,
+        SAFE_CAST(NULL AS STRING) AS settlement_status,
+        SAFE_CAST(NULL AS DATE) AS settlement_status_time,
+        SAFE_CAST(NULL AS STRING) AS settlement_reason_code,
+        SAFE_CAST(NULL AS STRING) AS settlement_response_text,
+
+        _line_number,
+        currency_code,
+        instance,
+        extract_filename,
+        ts,
+        littlepay_export_ts,
+        littlepay_export_date,
+        _content_hash,
+        _key,
+        _payments_key,
+        'micropayments' AS source_table
+
+    FROM {{ ref('stg_littlepay__micropayments') }}
+    WHERE type = 'CREDIT'
+        AND _key NOT IN ('043ecc000223a299ce17f6a342b1d240', '3536fb2035bbcf4dcb1f3abf001b5185')
```

Comment:
I would add a comment explaining that these are two that appear in both tables and it was easier to drop them manually than trying to make an overfit filter